iothub/service/src/Configurations/Models/ConfigurationImportMode.cs

```diff
@@ -18,7 +17,6 @@ public enum ConfigurationImportMode
         /// If the configuration already exists, existing information is overwritten with the provided input data only if there is an ETag match.
         /// If there is an ETag mismatch, an error is written to the log file.
         /// </summary>
-        [EnumMember(Value = "createOrUpdateIfMatchETag")]
```

Comment:
Neither the attribute nor the enum value are needed, when we use enums as their string values.